modules/yarn.lock

```diff
@@ -14771,6 +14772,15 @@ terser-webpack-plugin@^5.0.3:
     source-map "^0.6.1"
     terser "^5.3.8"
 
+terser@5.3.8:
+  version "5.3.8"
+  resolved "http://localhost:4873/terser/-/terser-5.3.8.tgz#991ae8ba21a3d990579b54aa9af11586197a75dd"
```

Comment:
This is the problem :point_up: :-(. Will fix it tomorrow.

Reply:
Simply breathtaking. One more thing to be on the lookout for in diffs.

![amazing](https://user-images.githubusercontent.com/7074/100644865-2a8efb80-333c-11eb-8b4a-24f4d0bc55ca.gif)

Reply:
I would love not to use the local npm registry, but sadly I have no other way to test this locally. And I don't want to publish a prerelease for every little change :-(.

However, I have changed my bash prompt so that I can see if I have my local npm registry active and I will create a little script to check this. Hopefully that will prevent this from happening again (in addition to human checks, of course) :crossed_fingers: .